vendor/engines/projects/spec/services/projects/order_details/param_updater_extension_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe Projects::OrderDetails::ParamUpdaterExtension do
```

Comment:
I feel like a controller spec that makes sure that the project_id gets saved would be nice to have.